src/Features/CSharp/Portable/Completion/CompletionProviders/DeclarationNameCompletionProvider.cs

```diff
@@ -50,28 +55,40 @@ internal override bool IsInsertionTrigger(SourceText text, int insertedCharacter
             foreach (var (name, kind) in recommendedNames)
             {
                 // We've produced items in the desired order, add a sort text to each item to prevent alphabetization
-                completionContext.AddItem(CreateCompletionItem(name, GetGlyph(kind, nameInfo.DeclaredAccessibility), sortValue.ToString("D8")));
+                completionContext.AddItem(CreateCompletionItem(name, GetGlyph(kind, nameInfo[0].DeclaredAccessibility), sortValue.ToString("D8")));
```

Comment:
Shouldn't this come from the corresponding info?

Reply:
Strictly speaking, yes, but in practice, no. If we're declaring a member, there's only one possible accessibility (and only one possible type). It's only in the out parameter case that we wind up with multiple infos, but even in that case, they all have the same accessibility. Will add a comment describing the situation.